packages/components/src/BrandMoment/_docs/BrandMoment.mdx

```diff
@@ -20,18 +19,24 @@ import * as BrandMoment from "./BrandMoment.stories"
 
 A brand moment is a page that showcases the brand's vibrant animations and illustration. The page provides an engaging experience for users when they reach these junction, helping to create a strong connection between the brand and product.
 
-<InlineNotification persistent type="informative">Code snippets on this page have been disabled due to performance issues, see [Source Code](https://github.com/cultureamp/kaizen-design-system/tree/main/packages/components/src/BrandMoment) for code examples.</InlineNotification>
+## Variant
 
-## Variants
+The `variant` prop sets the background color. Previously named `mood`.
```

Comment:
I'd say this is unnecessary here, I'd imagine the reader not caring about what it used to be

Reply:
Moreso because I'm not documenting the `mood` prop itself. Just for the people who do read 😬